### PR TITLE
add English version of #140

### PR DESCRIPTION
**Pull request by kmaehashi**
#140 document about connect method.

### CODE REVIEW

File: source/en/common_structs.rst [hunk 1]
```diff
@@ -188,3 +188,48 @@ Methods
   The common use case of this method is to close the TCP connection explicitly or to change the timeout.
 
   ``mprpc_client`` is a type of MessagePack-RPC client that is different between languages (`C++ <http://ci.jubat.us/job/msgpack-rpc/doxygen/classmsgpack_1_1rpc_1_1client.html>`_ / `Python <https://github.com/msgpack/msgpack-rpc-python/blob/master/msgpackrpc/client.py>`_ / `Ruby <http://msgpack.org/rpc/rdoc/current/MessagePack/RPC/Client.html>`_ / `Java <http://msgpack.org/rpc/javadoc/current/org/msgpack/rpc/Client.html>`_).
+
+
+Auxiliary Methods
+~~~~~~~~~~~~~~~~~
+
+Each language has some language-specific auxiliary methods.
```
Comment (gwtnb) [inline source/en/common_structs.rst]: "Each language has some..." seems that all client languages have auxiliary methods.

File: source/en/common_structs.rst [hunk 2]
```diff
@@ -188,3 +188,48 @@ Methods
...
+We'll describe them here.
```
Comment (gwtnb) [inline source/en/common_structs.rst]: This sentence seems verbose.

File: source/en/common_structs.rst [hunk 3]
```diff
@@ -188,3 +188,48 @@ Methods
...
+.. rb:module:: Jubatus::Common
+
+.. rb:class:: ClientBase
+
+   All client objects are defined as subclass of `ClientBase` class.
+
+   .. rb:classmethod:: connect(host, port, name, timeout_sec, &block)
+
+      Using `connect` method of client classes of each algorithms ensures safely closing client connections.
+      `connect` method takes host name, port number, cluster name, time-out period and block as arguments. It automatically creates client object and connect to the server specified.
```
Comment (gwtnb) [inline source/en/common_structs.rst]: a client object

**Comment by kmaehashi** [comment]
Thanks for comments!  All fixed.

File: source/en/common_structs.rst [hunk 4]
```diff
@@ -188,3 +188,47 @@ Methods
...
+.. rb:module:: Jubatus::Common
+
+.. rb:class:: ClientBase
+
+   All client objects are defined as subclass of `ClientBase` class.
+
+   .. rb:classmethod:: connect(host, port, name, timeout_sec, &block)
+
+      Using `connect` method of client classes of each algorithms ensures safely closing client connections.
+      `connect` method takes host name, port number, cluster name, time-out period and block as arguments. It automatically creates a client object and connect to the server specified.
```
Comment (gintenlabo) [inline source/en/common_structs.rst]: connect to the server → connects to the server

**Pull request merged**